stimela/backends/flavours/casa.py

```diff
@@ -59,10 +61,14 @@ def get_arguments(self, cab: Cab, params: Dict[str, Any], subst: Dict[str, Any],
                     casa_opts = [context.evaluate(opt, location=["opts"]) for opt in casa_opts]
                 except Exception as exc:
                     raise SubstitutionError(f"error substituting casa options '{casa_opts}'", exc)
-
-        # check for virtual_env
-        if virtual_env and "/" not in command:
-            command = f"{virtual_env}/bin/{command}"
+            wrapper = self.wrapper if self.wrapper is not None else casa_config.get('wrapper', 'xvfb-run -a')
+            if wrapper:
+                try:
+                    wrapper = [context.evaluate(wrapper, location=["wrapper"])]
```

Comment:
Sorry, @tmolteno, rather do ``wrapper = context.evaluate(wrapper, location=["wrapper"]).split()`` here.

Reply:
Cool, could you push the fix to this same branch then?